datapackage.json

```diff
@@ -0,0 +1,217 @@
+{
+  "name": "Unicode-Codes",
+  "title": "Unicode-Codes",
```

Comment:
Not sure if this is any better than `name` attribute. Titles are a little more human-readable than identifiers.